website/docs/r/docdb_cluster.html.markdown

```diff
@@ -0,0 +1,115 @@
+* `master_username` - (Required unless a `snapshot_identifier` is provided) Username for the master DB user. * `final_snapshot_identifier` - (Optional) The name of your final DB snapshot
+    when this DB cluster is deleted. If omitted, no final snapshot will be
+    made.
+* `skip_final_snapshot` - (Optional) Determines whether a final DB snapshot is created before the DB cluster is deleted. If true is specified, no DB snapshot is created. If false is specified, a DB snapshot is created before the DB cluster is deleted, using the value from `final_snapshot_identifier`. Default is `false`.
+* `availability_zones` - (Optional) A list of EC2 Availability Zones that
+  instances in the DB cluster can be created in
+* `backup_retention_period` - (Optional) The days to retain backups for. Default `1`
+* `preferred_backup_window` - (Optional) The daily time range during which automated backups are created if automated backups are enabled using the BackupRetentionPeriod parameter.Time in UTC
+Default: A 30-minute window selected at random from an 8-hour block of time per region. e.g. 04:00-09:00
+* `preferred_maintenance_window` - (Optional) The weekly time range during which system maintenance can occur, in (UTC) e.g. wed:04:00-wed:04:30
+* `port` - (Optional) The port on which the DB accepts connections
+* `vpc_security_group_ids` - (Optional) List of VPC security groups to associate
+  with the Cluster
+* `snapshot_identifier` - (Optional) Specifies whether or not to create this cluster from a snapshot. You can use either the name or ARN when specifying a DB cluster snapshot, or the ARN when specifying a DB snapshot.
+* `storage_encrypted` - (Optional) Specifies whether the DB cluster is encrypted. The default is `false` for `provisioned` `engine_mode` and `true` for `serverless` `engine_mode`.
+* `apply_immediately` - (Optional) Specifies whether any cluster modifications
+     are applied immediately, or during the next maintenance window. Default is
+     `false`.
+* `db_subnet_group_name` - (Optional) A DB subnet group to associate with this DB instance.* `db_cluster_parameter_group_name` - (Optional) A cluster parameter group to associate with the cluster.
+* `kms_key_id` - (Optional) The ARN for the KMS encryption key. When specifying `kms_key_id`, `storage_encrypted` needs to be set to true.
+* `engine` - (Optional) The name of the database engine to be used for this DB cluster. Defaults to `docdb`. Valid Values: `docdb`
+* `engine_version` - (Optional) The database engine version. Updating this argument results in an outage.
+* `enabled_cloudwatch_logs_exports` - (Optional) List of log types to export to cloudwatch. If omitted, no logs will be exported.
+   The following log types are supported: `audit`.
+* `tags` - (Optional) A mapping of tags to assign to the DB cluster.
+
+## Attributes Reference
+
+In addition to all arguments above, the following attributes are exported:
```

Comment:
The below list should be pared down to only include additional attributes and alphabetized for clarity.